backend/src/flotilla/api/authentication.py

```diff
@@ -24,6 +24,7 @@ def __init__(
         if self.authentication_enabled:
             self.scheme = SingleTenantAzureAuthorizationCodeBearer(
                 app_client_id=self.app_client_id,
+                token_version=1,
```

Comment:
Are you sure this shouldn't be version 2?

Reply:
Version 2 doesn't work with audience. It does not (?) add the "api://" before the scope or something; a common problem it seems